### ISSUE TITLE
Error signs in the gutter move my code to the right

### ISSUE THREAD

**Issue by giluis** [neovim/nvim-lspconfig#1309]
### Description

While programming in Typescript (Deno), whenever I get errors, I see a red "E" in the gutter. 
This is fine, except this E "pushes" my code to the right (screenshots below)  
Is this normal behaviour? If so, how do I disable it?
I like the red "E", I just don't want my code to move whenever there is an error.
I am running the default denols configuration that is on this repo's README.md
Let me know if this is not the right place to ask this question. 
Thank for your attention.

No error:
![image](https://user-images.githubusercontent.com/52404649/137473029-ff8d96ef-63c0-48e0-886d-0af248386bd4.png)

Error (code moved 1 character to the right): 
![image](https://user-images.githubusercontent.com/52404649/137473093-7b1e8214-305f-4d73-8d81-1bfd53d7d708.png)

### Neovim version

NVIM v0.6.0-dev
Build type: RelWithDebInfo
LuaJIT 2.1.0-beta3

### Operating system and version

Ubuntu 20.04

### Affected language servers

denols

### Steps to reproduce

1. install nvm-lspconfig
2. require'lspconfig'.denols.setup{} in init.vim
3. nvim somefile.ts

### Actual behavior

Is moved to the right when there is an error, comes back to original spot when there are no errors

### Expected behavior

Should not move my code to display the red "E" 

### Minimal config

```Lua
Just added the following to the bottom of the minimal_init.lua

require'lspconfig'.denols.setup{}
```

### Health check

lspconfig: require("lspconfig.health").check()
======================================================================
  - INFO: denols: configuration checked.

### LSP log

I'm using minimal_init.lua

**Comment by clason** [comment]
Yes, this is normal -- see `:h 'signcolumn'`. If you don't want columns to be inserted or removed, set that to `yes`.

(For such questions, there is a separate link to the [Neovim Discourse Group](https://neovim.discourse.group) in the new issue dialog. You can also ask on https://matrix.to/#/#neovim:matrix.org.)
